src/routes/gallery.svelte

```diff
@@ -1,23 +1,17 @@
 <script>
+	import HouseCarousel from '$lib/components/Carousel/houseCarousel.svelte';
+	import LocalAreaCarousel from '$lib/components/Carousel/localAreaCarousel.svelte';
 	import Container from '$lib/components/container.svelte';
 </script>
 
 <Container>
 	<h1>Gallery</h1>
 	<h2>Pictures of the accommodation</h2>
 
-	<div class="flex justify-around bg-violet-500">
-		<img
-			class="object-scale-down h-50 w-96"
-			src="house.jpeg"
-			alt="Front view of the house"
-		/>
-		<img
-			class="object-scale-down h-50 w-96"
-			src="rustic.jpeg"
-			alt="Rustic accommodation view"
-		/>
-	</div>
+	<HouseCarousel />
```

Comment:
The carousel is generated in a bespoke component, to add a new carousel you can copy & paste a component, give it a different name & ID and add the photos you'd like